src/pages/box-shadow.ts

```diff
@@ -35,7 +36,7 @@ const attribute = 'box-shadow';
 let isSliderOpen = false;
 const getCssOrTailwindDropdownElement = getCssOrTailwindDropdown(attribute);
 const showCopyClass = 'show-css-tailwind';
-
+const getResultBtn = document.getElementById("getResultBtn");
```

Comment:
As i said, there's already a away to get with this function
https://github.com/Dun-sin/Code-Magic/blob/506f88d887ab56e9a1a9b8114680270578624de9/src/lib/getElements.ts#L42